src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,121 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    public Book() {
+    }
+
+    public Book(String genre, String author, String title, String subtitle, String publisher, String year, Integer pages, String isbn) {
+        this.genre = genre;
+        this.author = author;
+        this.title = title;
+        this.subtitle = subtitle;
+        this.publisher = publisher;
+        this.year = year;
+        this.pages = pages;
+        this.isbn = isbn;
+    }
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
You should use `GenerationType.SEQUENCE`. This is because Postgres use `GenerationType.AUTO` as global level

Reply:
nice to know